generate-sol-log.sh

```diff
@@ -13,7 +13,7 @@ bmc_account_list="${BMC_ACCOUNT_LIST}"
 handled_ip_list=""
 vnode_num=0
 timeout=0
-maxto=50
+maxto=20
```

Comment:
In production env I find that 50 times is redundant, sometimes when smoke-test ends this script is still running.
After assessment I think 20 is suitable.